libraries/dynamic-ui-react/src/style/abstracts/variables/_forms.scss

```
@@ -236,7 +241,7 @@ $form-range-track-height: .5rem !default;
 $form-range-track-cursor: pointer !default;
 $form-range-track-bg: var(--#{$prefix}gray-300) !default;
 $form-range-track-border-radius: 1rem !default;
-$form-range-track-box-shadow: $box-shadow-inset !default;
+
```

por acá habría que dejar la definición que propone bootstrap, como comento arriba. 

```suggestion
$form-range-track-box-shadow: var(--#{$prefix}box-shadow-inset) !default;
```